can-validate-validatejs.md

```diff
@@ -1,21 +1,71 @@
 @module {Object} can-validate-validatejs
 @parent can-ecosystem
-@group can-validate-validatejs.core 1 can-validate-validatejs Core
 @package ./package.json
 
-@description A plugin for CanJS that wraps any validation library to `can.validate`.
-**Can-Validate doesn't do any validation of its own** but instead provides some abstraction to your library of choice. The chosen library is registered with can-validate using a shim.
+@description Create [can-validate/validator] functions using [validate.js](https://validatejs.org/).
```

Comment:
`[can-validate/validator]` doesn’t seem to render correctly when I generate the docs:
<img width="563" alt="screen shot 2017-01-12 at 12 00 53 pm" src="https://cloud.githubusercontent.com/assets/10070176/21906270/74adff12-d8bf-11e6-9983-9fc436709d51.png">